plugins/backstage-plugin-coder/src/testHelpers/setup.tsx

```diff
@@ -138,76 +141,6 @@ export const CoderProviderWithMockAuth = ({
   );
 };
 
-type ChildProps = EntityProviderProps;
```

Comment:
You can ignore this entire red section – these were bad test helpers that didn't do what they were supposed to. They're just deleted – I added a new test helper towards the bottom of the file